Crash with 1.3.0

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{org.glucosio.android/org.glucosio.android.activity.HelloActivity}: java.lang.StringIndexOutOfBoundsException: length=3; regionStart=0; regionLength=5
    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2665)
    at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2726)
    at android.app.ActivityThread.-wrap12(ActivityThread.java)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1477)
    at android.os.Handler.dispatchMessage(Handler.java:102)
    at android.os.Looper.loop(Looper.java:154)
    at android.app.ActivityThread.main(ActivityThread.java:6119)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
Caused by: java.lang.StringIndexOutOfBoundsException: length=3; regionStart=0; regionLength=5
    at java.lang.String.substring(String.java:1931)
    at org.glucosio.android.tools.LocaleHelper.java.lang.String getDisplayLanguage(java.lang.String)(Unknown Source)
                                            void updateLanguage(android.content.Context,java.lang.String)
                                            java.util.List getLocalesWithTranslation(android.content.res.Resources)
                                            java.util.Locale getDeviceLocale()
    at org.glucosio.android.activity.HelloActivity.void initLanguageSpinner(org.glucosio.android.tools.LocaleHelper)(Unknown Source)
                                                void setSelection(java.lang.String,org.glucosio.android.tools.LabelledSpinner)
    at org.glucosio.android.activity.HelloActivity.void onCreate(android.os.Bundle)(Unknown Source)
    at android.app.Activity.performCreate(Activity.java:6679)
    at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1118)
    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2618)
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,8 +6,8 @@ jdk:
 android:
   components:
     - tools
-    - build-tools-24.0.0
-    - android-24
+    - build-tools-25.0.0
+    - android-25
     - extra-android-support
     - extra-google-google_play_services
     - extra-android-m2repository

--- a/app/src/main/java/org/glucosio/android/tools/LocaleHelper.java
+++ b/app/src/main/java/org/glucosio/android/tools/LocaleHelper.java
@@ -66,7 +66,7 @@ public class LocaleHelper {
         Collections.addAll(translatedLanguageSet, translatedLanguages);
 
         for (String language : languageList) {
-            if (translatedLanguageSet.contains(language.substring(0, 5)))
+            if (translatedLanguageSet.contains(language))
                 availableLanguagesSet.add(language);
         }
 

